integreat_cms/core/signals/hix_signals.py

```diff
@@ -20,33 +20,31 @@ def page_translation_save_handler(instance, **kwargs):
     :param \**kwargs: The supplied keyword arguments
     :type \**kwargs: dict
     """
-    if kwargs.get("raw"):
+
+    if kwargs.get("raw") or instance.hix_score:
```

Comment:
```suggestion
    if kwargs.get("raw"):
```
This looks like a good optimization at first glance, but will often times cause incorrect HIX values.
Whenever we create new versions, we inherit all fields (including the HIX value) from the old version, and only overwrite the content manually. So we have to re-calculate the HIX score here, and can only keep it if the content is actually identical (thus the check for `identical_version` further below)...